conmon/conmon.c

```diff
@@ -128,6 +134,7 @@ static GOptionEntry opt_entries[] = {
 	{"exit-command-arg", 0, 0, G_OPTION_ARG_STRING_ARRAY, &opt_exit_args,
 	 "Additional arg to pass to the exit command.  Can be specified multiple times", NULL},
 	{"log-path", 'l', 0, G_OPTION_ARG_STRING, &opt_log_path, "Log file path", NULL},
+	{"log-to-journald", 0, 0, G_OPTION_ARG_NONE, &opt_log_to_journald, "Log to journald as wel as log-path", NULL},
```

Comment:
I am not a big fan of having a different option, I'd rather prefer we support multiple `--log-path`, see comment:

https://github.com/kubernetes-sigs/cri-o/pull/2078#discussion_r260213363